i18n/en.json

```diff
@@ -91,8 +91,10 @@
 	"pager-older-n": "{{PLURAL:$1|Older $1|Older $1}}",
 	"pages-created": "Pages created",
 	"pages-created-desc": "A content page is a page in either the main or file namespace.",
+	"pages-created-wikidata": "Items created",
 	"pages-improved": "Pages improved",
 	"pages-improved-desc": "A content page is a page in either the main or file namespace.",
+	"pages-improved-wikidata": "Items improved",
```

Comment:
don't forget qqq :)

Reply:
Good point! Should we think about adding a check for that to the CI? A codesniff should be added for it to Intuition maybe?

Reply:
That would be magnificent! If you know how to do it, by all means :) I wasn't sure if you could easily program in your own sniff rules like that.

Reply:
That would be magnificent! If you know how to do it, by all means :) I wasn't sure if you could easily program in your own sniff rules like that.